docs/application/dotnet/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,506 @@
+
+4. To pause a media source, use the `Pause` property:
+
+    ```csharp
+    mediaSource.Pause = true;
+    ```
+    > [!NOTE]
+    > Pausing a media source means it does not send the data to a remote peer. Pause or resume of a media source is also possible in `WebRTCState.Playing` state.
+    > `MediaNullSource` does not support this functionality.
+
+5. To mute a media source, use the `Mute` property:
+
+    ```csharp
+    mediaSource.Mute = true;
+    ```
+    > [!NOTE]
+    > Muting a media source means it sends black video frames or silent audio frames to a remote peer. Muting or unmuting of a media source is also possible in the `WebRTCState.Playing` state.
+    > Some types of media sources do not support this functionality, such as, `MediaFileSource`, `MediaPacketSource`, and `MediaNullSource`.
+
+6. To set or get the encoder bitrate to a media source, use the `EncoderBitrate` property:
+
+    ```csharp
+    ...
+    await webRtc.StartAsync();
+    ...
+    var encoderBitrate = mediaSource.EncoderBitrate;
+    // some logic to increase or decrease the current target bitrate
+    // e.g. var newEncoderBitrate = encoderBitrate / 2;
+    mediaSource.EncoderBitrate = newEncoderBitrate;
+    ```
+    > [!NOTE]
+    > Some types of media sources do not support this functionality, such as, `MediaFileSource`, `MediaPacketSource`, and `MediaNullSource`.
+
+7. To set or get the video resolution to a media source, use the `VideoResolution` property:
+
+    ```csharp
+    mediaSource.VideoResolution = new Size(640, 480);
+    ```
+    > [!NOTE]
+    > Some types of media sources support dynamic resolution change while streaming. Otherwise the `InvalidOperationException` exception will be thrown.
+
+8. To set or get the video frame rate to a media source, use the `VideoFrameRate` property:
+
+    ```csharp
+    mediaSource.VideoFrameRate = 15;
+    ```
+    > [!NOTE]
+    > If the input value is not supported by the media source, the `ErrorOccurred` event will be invoked.
+
+9. To set the device ID to a media source for camera, use the `CameraDeviceId` property before calling `Start()` or `StartAsync()`:
+
+    ```csharp
+    ...
+    var cameraSource = new MediaCameraSource();
+    webRtc.AddSource(cameraSource);
+    ...
+    cameraSource.CameraDeviceId = 1;
+    ...
+    await webRtc.StartAsync();
+    ```
+    > [!NOTE]
+    > If the device ID is not valid, `StartAsync()` or `Start` will throw `InvalidOperationException` exception.
+
+<a name="data_channel"></a>
+## Control data channels
+
+You can create a data channel to the WebRTC. It is also possible to be notified when you have a new data channel requested by a remote peer. You can send, or receive data, to or from these data channels by using the APIs below.
+
+1. To create a data channel, use `WebRTCDataChannel` before calling `Start()` or `StartAsync()`:
```

Comment:
Hello @hsgwon , I have a question in Line# 187. Is the event `WebRTCDataChannel` in the correct form? Should I be `WebRTC.DataChannel` instead?
I hope you address my confusion here, thank you.

Reply:
`WebRTCDataChannel` is not a event, it's class. 
Do I need to add a word `class` after `WebRTCDataChannel`?

Reply:
Thank you for the clarification. Yes, I think adding the word **class** after **WebRTCDataChannel** would make it clearer, so I'd suggest you do that. Thank you.

Reply:
OK I'll add `class` after `WebRTCDataChannel`.